### PR TITLE
Slip 77 derivation

### PR DESCRIPTION
**Pull request by altafan**
This closes [#76](https://github.com/vulpemventures/liquid-js-monorepo/issues/76).
This closes [#77](https://github.com/vulpemventures/liquid-js-monorepo/issues/77).
This closes [#78](https://github.com/vulpemventures/liquid-js-monorepo/issues/78).

NOTE: the only one test case that I found is taken from libwally and refers to the one described in SLIP-21.
Since the derivation scheme of the master follows SLIP-77 you should not consider the expected values in SLIP-21 as the expected values of our unit test.

### CODE REVIEW

File: ts_src/index.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  }
+}
+
+export function fromMasterBlindingKey(key: Buffer | string): Slip77Interface {
+  typeforce(typeforce.anyOf('Buffer', 'String'), key);
+  const masterKey = Buffer.isBuffer(key) ? key : Buffer.from(key, 'hex');
+  return new Slip77(masterKey, undefined, undefined, undefined, undefined);
+}
+
+export function fromBip39Seed(_seed: Buffer | string): Slip77Interface {
```
Comment (tiero) [inline ts_src/index.ts]: I would call just fromSeed

File: ts_src/index.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    private _extra: Buffer | undefined,
+    private _script: Buffer | undefined,
+    private _privateKey: Buffer | undefined,
+    private _publicKey: Buffer | undefined,
+  ) {
+    typeforce(typeforce.BufferN(32), _masterKey);
+  }
+
+  get masterKey(): Buffer {
+    return this._masterKey;
```
Comment (tiero) [inline ts_src/index.ts]: I would not use the underscore

Comment (altafan) [reply ts_src/index.ts]: How would I not use the underscore? bip32 does exactly the same using 2 underscores

File: ts_src/index.ts (hunk 1 above)
Comment (tiero) [reply ts_src/index.ts]: @altafan

File: README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Example
+
+- [Generate a master blinding key from BIP39 seed](./tests/index.js#L20)
+- [Import a master blinding key](./tests/index.js#L31)
+- [Derive a blinding key pair](./tests/index.js#L39)
+
+## LICENSE [MIT](LICENSE)
+
+A derivation (and extraction for modularity) of the `HDWallet`/`HDNode` written and tested by [liquidjs-lib](https://github.com/vulpemventures/liquidjs-lib) contributors since 2020.
```
Comment (tiero) [inline README.md]: not relevant, remove

Comment (tiero) [reply README.md]: @altafan

**Pull request merged**